modules/update-custom-certificate-central.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="update-custom-certificate-central_{context}"]
+= Updating the custom certificate for an existing instance
+
+[role="_abstract"]
+If you are using a custom certificate for Central, you can update it by following the instructions in this section.
+
+.Procedure
+. Delete the existing custom certificate's secret.
++
+[source,terminal]
+----
+$ oc delete secret central-default-tls-cert
```

Comment:
Is `central-default-tls-cert` the standard name for all installations? Or is this an example, and users must supply their own secret name here?

If it is an example, I'd suggest making that explicit.

Reply:
Thank you @ctauchen 

Those are the default secret names that RHACS generates.